internal/plugin_config_test.go

```diff
@@ -1127,3 +1129,35 @@ func TestPolicySetConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestDisabled(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	createConfigMap(t, tmpDir, "configmap.yaml")
+	configMapPath := path.Join(tmpDir, "configmap.yaml")
+	defaultsConfig := fmt.Sprintf(
+		`
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: policy-generator-name
+policyDefaults:
+  namespace: my-policies
+  disabled: true
+policies:
+- name: policy-app-config
```

Comment:
Could you please add another policy that overrides the `disabled` field from the defaults? Then just check to make sure the override is respected.

Reply:
@mprahl added one